utils.test.js

```diff
@@ -31,5 +32,8 @@ describe('utils', function() {
   it('checkUrl', () => {
     expect(checkUrl(['wiki.jenkins.io'], 'https://wiki.jenkins.io/something')).toBe(true);
   });
+  it('removeExternalLink', () => {
+    expect(removeExternalLink("(https://github.com/jenkinsci/plugin-pom){.external-link}.")).toBe("(https://github.com/jenkinsci/plugin-pom)");
```

Comment:
@halkeye I am not too sure how to check for adoc and MD, is there any resource I can refer to.

Reply:
Sorry I'm a bit distracted by a few others projects, being sick, and work thing

But i'm paying attention I swear :)

I don't think anything comes to mind. Maybe find a plugin where this is happening, and export both an ascii and a markdown file, then add those both as strings/files for the tests?

Reply:
I will try and give it a shot

Reply:
Yeah I added an markdown file for test purposes